levelkdev/swapr-dapp
# fix: fix infinite loop update in useAutoMaxBalance hook

# Summary

Fixes #1240

Fix infinite loop update in useAutoMaxBalance hook.

  # To Test

1. Open the page `swap`
2. Click on max balance
3. Choose another token
- [ ] App doesn't crash
- [ ] New token max balance should be displayed
